tools/vol2obj/main.c

```diff
@@ -505,7 +508,16 @@ int main( int argc, char** argv ) {
     }
     strncat( dad_hdr_str, "header.vols", 2047 );
     strncat( dad_seq_str, "sequence_0.vols", 2047 );
-    strncat( dad_vid_str, "texture_2048_h264.mp4", 2047 );
+    // Try to use a 2k texture if one is in the folder, otherwise go for 1024x1024.
+    strncat( test_vid_str, dad_vid_str, 2047 );
+    strncat( test_vid_str, VOL_VID_STR_2048, 2047 );
+    FILE* ft_ptr = fopen( test_vid_str, "rb" );
+    if ( !ft_ptr ) {
+      strncat( dad_vid_str, VOL_VID_STR_1024, 2047 );
+    } else {
+      fclose( ft_ptr );
+      strncat( dad_vid_str, VOL_VID_STR_2048, 2047 );
+    }
```

Comment:
(maybe stupid question) can/makes sense `2047` be extracted to a const or similar?

Reply:
yes i will change that

Reply:
done